src/elf/reloc.rs

```diff
@@ -183,12 +189,9 @@ macro_rules! elf_rela_std_impl { ($size:ident, $isize:ty) => {
 
             pub fn from_fd(fd: &mut File, offset: usize, size: usize) -> Result<Vec<Rela>> {
                 let count = size / SIZEOF_RELA;
-                let mut bytes = vec![0u8; size];
+                let mut relocs = vec![Rela::default(); count];
```

Comment:
All these buffer allocations could be very slightly more efficient if `plain` also provided a function for allocating uninitialized vectors of `Plain` type. Probably not worth the trouble though.